ambari-server/src/main/java/org/apache/ambari/server/controller/internal/ActiveWidgetLayoutResourceProvider.java

```diff
@@ -252,6 +253,14 @@ public Void invoke() throws AmbariException, AuthorizationException {
     return getRequestStatus(null);
   }
 
+  private String getUserName(Map<String, Object> propertyMap) {
+    String userName = propertyMap.get(WIDGETLAYOUT_USERNAME_PROPERTY_ID) == null ? "" : propertyMap.get(WIDGETLAYOUT_USERNAME_PROPERTY_ID).toString();
```

Comment:
`propertyMap.get(WIDGETLAYOUT_USERNAME_PROPERTY_ID)` could be extracted to a local variable.

Reply:
Could be :)